tests/suites/1_stateful/02_query/02_0010_udf.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+echo "SET external_server_request_max_threads=1; SELECT ping(number) FROM numbers(100000) IGNORE_RESULT;" | $BENDSQL_CLIENT_CONNECT
```

Comment:
let's also decrease the setting of `batch_rows `

echo "SET external_server_request_batch_rows = 1024; SET external_server_request_max_threads=1; SELECT ping(number) FROM numbers(100000) IGNORE_RESULT;" | $BENDSQL_CLIENT_CONNECT